sources/packages/web/src/views/aest/SearchStudents.vue

```diff
@@ -16,7 +16,7 @@
               variant="outlined"
               v-model="appNumber"
               @keyup.enter="searchStudents"
-              hide-details
+              hide-details="auto"
```

Comment:
We centralized the validations in the `v-input`, do we need auto `hide-details="auto"` here?